backend/package.json

```diff
@@ -34,6 +34,7 @@
     "bcrypt": "^3.0.8",
     "class-transformer": "^0.2.3",
     "class-validator": "^0.10.2",
+    "moment": "^2.24.0",
```

Comment:
This is apparently not used. I will update pr later with removal of this, trace of code before